src/components/SlateEditor/plugins/related/EditRelated.jsx

```diff
@@ -145,7 +136,75 @@ class EditRelated extends React.PureComponent {
               <h1 className="c-section-heading c-related-articles__component-title">
                 {t('form.related.title')}
               </h1>
-              <StyledListWrapper>{relatedArticles}</StyledListWrapper>
+              <p>{t('form.related.subtitle')}</p>
+              <StyledListWrapper>
+                <DragDropContext
+                  onDragEnd={a => {
+                    const toIndex = a.destination.index;
+                    const fromIndex = a.source.index;
+                    const newArticles = [...articles];
+
+                    var element = newArticles[fromIndex];
+                    newArticles.splice(fromIndex, 1);
+                    newArticles.splice(toIndex, 0, element);
+                    updateArticles(newArticles);
+                  }}>
+                  <Droppable droppableId="relatedArticleDroppable">
+                    {(provided, snapshot) => (
+                      <div
+                        ref={provided.innerRef}
+                        className={
+                          snapshot.isDraggingOver
+                            ? 'drop-zone dragging'
+                            : 'drop-zone'
+                        }>
+                        {articles.map((article, index) => {
+                          if (!article) {
+                            return null;
+                          }
+                          return (
+                            <Draggable
+                              key={article.id}
+                              draggableId={article.id}
+                              index={index}>
+                              {(providedInner, snapshotInner) => (
+                                <div
+                                  className="drag-item"
+                                  ref={providedInner.innerRef}
+                                  {...providedInner.dragHandleProps}
+                                  {...providedInner.draggableProps}>
+                                  <StyledArticle
+                                    isDragging={snapshotInner.isDragging}
+                                    dragHandleProps={
+                                      providedInner.dragHandleProps
+                                    }
+                                    key={article.id}>
+                                    <RelatedArticle item={article} />
+                                    <DeleteButton
+                                      title="Fjern relaterte artikler"
+                                      stripped
+                                      onClick={e => {
```

Comment:
syns det er mer oversiktlig om denne taes ut i en egen funksjon

Reply:
Jeg syntes det var et rart valg selv.

Om du ønsker å vite hva denne funksjonen gjør så er det vel enklere å lese en 5 linjers funksjon enn å scrolle opp til en definisjon som ligger et annet sted?

Om jeg skulle skilt ut noe her så hadde jeg valgt å skille ut funksjonen som mapper artikler til draggables.

Hva gjorde at den funksjonen stakk ut for deg?

Reply:
Kan være enig med Anders her. Om funksjonen kun brukes her og heller ikkje skal testes utenfra er det meir lesbart å ha den inline. 
Alternativt kan den ha et navn som gjør at du ikkje treng å lese definisjonen. removeSelectedArticleFromList.